GEOSldas_App/util/shared/python/read_GEOSldas.py

```diff
@@ -0,0 +1,239 @@
+            param = {}
+            param['descr']          =       fid.readline().strip().strip('"')
+            param['species']        = float(fid.readline().strip())
+            param['orbit']          = float(fid.readline().strip())
+            param['pol']            = float(fid.readline().strip())
+
+            param['N_ang']          = int(float(fid.readline().strip()))
+
+            param['ang']            = np.array([float(x) for x in fid.readline().split()])
+
+            param['freq']           = float(fid.readline().strip())
+            param['FOV']            = float(fid.readline().strip())
+            param['FOV_units']      =       fid.readline().strip().strip('"')
+            param['assim']          =       fid.readline().strip()
+            param['scale']          =       fid.readline().strip()
+            param['getinnov']       =       fid.readline().strip()
+            param['RTM_ID']         = float(fid.readline().strip())
+            param['bias_Npar']      = float(fid.readline().strip())
+            param['bias_trel']      = float(fid.readline().strip())
+            param['bias_tcut']      = float(fid.readline().strip())
+            param['nodata']         = float(fid.readline().strip())
+            param['varname']        =       fid.readline().strip().strip('"')
+            param['units']          =       fid.readline().strip().strip('"')
+            param['path']           =       fid.readline().strip().strip('"')
+            param['name']           =       fid.readline().strip().strip('"')
+            param['maskpath']       =       fid.readline().strip().strip('"')
+            param['maskname']       =       fid.readline().strip().strip('"')
+            param['scalepath']      =       fid.readline().strip().strip('"')
+            param['scalename']      =       fid.readline().strip().strip('"')
+            param['flistpath']      =       fid.readline().strip().strip('"')
+            param['flistname']      =       fid.readline().strip().strip('"')
+            param['errstd']         = float(fid.readline().strip())
+            param['std_normal_max'] = float(fid.readline().strip())
+            param['zeromean']       =       fid.readline().strip()
+            param['coarsen_pert']   =       fid.readline().strip()
+            param['xcorr']          = float(fid.readline().strip())
+            param['ycorr']          = float(fid.readline().strip())
+            param['adapt']          = float(fid.readline().strip())
+
+            obs_param.append(param)
+
+    print(f"Done reading obs_param for {N_obs_param} species")
+
+    return obs_param
+
+# ----------------------------------------------------------------------------
+#
+# reader for GEOSldas tilecoord file (binary)
+
+def read_tilecoord(fname):
+    int_precision   = 'i'
+    float_precision = 'f'
+
+    # SPECIFY endianness
+    machfmt = '<'                    # '>' for big-endian, '<' for little-endian
```

Comment:
could/should this be moved to the top of the "module" (global variable for the module, in Fortran speak)?  It's rather hidden in the middle of things here. Note that the same occurs in the next function.

Reply:
I am not sure if we can do that without making a class. Let's keep it as is for now.